### PR TITLE
feat: product listing UI

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added filter component
- added product card component
- added data to the api db

### CODE REVIEW

File: src/pages/product/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+const Filter = () => {
+    const [category, setCategory] = useState([]);
+
+    const getData = async () => {
+        try {
+            const response = await axios.get("/api/categories");
```
Comment (JantuDeb) [inline src/pages/product/Filter.jsx:10]: destructure `data` from response

File: src/pages/product/Product.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { AiFillHeart } from "react-icons/ai";
+
+const Product = () => {
+
+    const [item, setItem] = useState([]);
```
Comment (JantuDeb) [inline src/pages/product/Product.jsx]: Variable name could be `[products,setProducts]`

**Review by JantuDeb — APPROVED** [review]
Left few comments, overall the codes looks fine.

File: src/pages/product/Filter.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+const Filter = () => {
+    const [category, setCategory] = useState([]);
+
+    const getData = async () => {
```
Comment (rkapoor10) [inline src/pages/product/Filter.jsx:8]: jas...getCategories

File: src/pages/product/Product.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+    const getData = async () => {
```
Comment (rkapoor10) [inline src/pages/product/Product.jsx]: jas...getProducts()

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge.

File: src/pages/product/Product.jsx (hunk 2 above)
Comment (harshitadaswani) [reply src/pages/product/Product.jsx]: done

File: src/pages/product/Filter.jsx (hunk 1 above)
Comment (harshitadaswani) [reply src/pages/product/Filter.jsx:10]: okay done

File: src/pages/product/Filter.jsx (hunk 3 above)
Comment (harshitadaswani) [reply src/pages/product/Filter.jsx:8]: done changed

File: src/pages/product/Product.jsx (hunk 4 above)
Comment (harshitadaswani) [reply src/pages/product/Product.jsx]: done changed

**Pull request merged**